corehq/apps/app_manager/templates/app_manager/partials/define_case_type_modal.html

```diff
@@ -0,0 +1,80 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<div
+  class="modal fade define-case-type-modal"
+  tabindex="-1"
+  role="dialog"
+  id="define-case-type-modal"
+>
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header" style="border-bottom: 1px solid lightgray;">
+        <button
+          type="button"
+          class="close"
+          data-dismiss="modal"
+          aria-label="Close"
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <h4 class="modal-title">
+          <i class="fa-solid fa-list-check"></i> {% trans "Case List" %}
+        </h4>
+      </div>
+      <div class="modal-body">
+        <h4>
+          <strong
+            >{% trans "Please define a case type for your case list." %}</strong
+          >
+        </h4>
+        <div class="form-horizontal" style="margin-top: 30px;">
+          <div class="form-group" id="new_case_type_form_group">
+            <label class="control-label col-sm-3">
+              {% trans "Case Type" %}
+              {% if request|toggle_enabled:'USH_CASE_CLAIM_UPDATES' %}
+                <span
+                  class="hq-help-template"
+                  data-title="{% trans "Case Type" %}"
+                  data-content="{% trans "The name for the type of case you are tracking (e.g. “pregnancy,” “farm,” “household,” etc.). All cases you register in this case list will be this type of case. Other case lists with the same case type will use the same set of case(s)." %}"
+                ></span>
+              {% else %}
+                <span
+                  class="hq-help-template"
+                  data-title="{% trans "Case Type" %}"
+                  data-content="{% trans "The name for the type of case you are tracking (e.g. “pregnancy,” “farm,” “household,” etc.). All cases you register in this case list will be this type of case, and only cases of this type will show in the case list. Other case lists with the same case type will use the same set of cases." %}"
+                ></span>
+              {% endif %}
+            </label>
+            <div class="new-case-type-dropdown-container col-sm-6">
+              <select
+                class="form-control new-case-type-dropdown"
```

Comment:
This also appears to be an unused class

Reply:
13ae7b64894a3d6d0a6ca66f40a98d08adaeb6a5